README.md

```diff
@@ -1,2 +1,38 @@
-# aragon-manager
-deploy and manage osx plugins with ease
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
+
+# Aragon Manager
+
+## Getting Started
+
+This project is intended to be used with the latest Active LTS release of [Node.js][nodejs].
+
+To clone the repository, use the following commands:
+
+```sh
+git clone https://github.com/aragon/aragon-manager
+cd aragon-manager
+pnpm install
+```
+
+In one terminal run build in watch mode 
```

Comment:
"In one terminal, run the build in watch mode through using the following command:"